### ISSUE TITLE
🐛 BUG: When a user profile link is in the body of a post, hovering causes fatal error

### ISSUE THREAD

**Issue by siliconmeadow** [dailydotdev/daily#1571]
### What went wrong? 🤔

When viewing [this post](https://app.daily.dev/posts/we-made-daily-dev-faster-see-how-we-did-it-uzorvcrrs) in a browser this morning, I hovered over the @idoshamun profile link in the body of the post and immediately saw:

![image](https://github.com/user-attachments/assets/e6d79ca2-6a7b-4779-8da6-721f877cc94b)

This only appeared when visiting the post in full view, rather than the modal view or when viewing it on the mobile app.

I spotted it in the Zen Browser (FF based) and Chrome Canary both on Arch Linux using the latest Hyprland compositor. I've not yet tried anything else other than the app on my android phone (which works fine, as I mentioned).

The console message that MuaazH [mentions here](https://dly.to/5gIMeYYziH8), also appears in my browser, but I'm not a frontender and the mention of NextJS and the minimised js files make my palms sweat and that I'm not likely to provide much further help solving the problem technically.

### Expected Behavior

When a user hovers over a profile link in a post, they should be able to continue to read the post instead of causing a fatal error.

### Steps to Reproduce Issue

```bash
There are two posts that I know of where you can reproduce the issue:

- The first I'd spotted it: https://app.daily.dev/posts/we-made-daily-dev-faster-see-how-we-did-it-uzorvcrrs
- One that I created here: https://app.daily.dev/posts/zPrVtpxdl#c-1TN5RtoNy

1. Go to either of the links above in a desktop browser and hover over the @<profile> link
2. See the error
```

### Environment

Arch linux w/Hyprland compositor & Zen Browser v1.0.1-a.17 or Chome Canary 131.0.6778.24 (Official Build) beta (64-bit) 

### Browsers

Chrome, Other

### OS

Linux

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by rebelchris** [comment]
Thanks for reporting!
We are actively trying to fix this asap 🙏

**Comment by ilasw** [comment]
Hi @siliconmeadow , thanks for reporting again, we have merged #3761 that contains a fix for this issue! ✅

**Comment by siliconmeadow** [comment]
> Hi @siliconmeadow , thanks for reporting again, we have merged [dailydotdev/apps#3761](https://github.com/dailydotdev/apps/pull/3761) that contains a fix for this issue! ✅

Hi @ilasw - I just had a look at the PR and code review comments and am really impressed by the effort you all have put into that. And it really looks good now too! Well done!
